tool_collections/kraken/kraken/kraken.xml

```diff
@@ -9,7 +9,23 @@
     <command>
         <![CDATA[
         kraken --threads \${GALAXY_SLOTS:-1} @INPUT_DATABASE@ 
+        
+        #if isinstance( $input_sequences.datatype, $__app__.datatypes_registry.get_datatype_by_extension('fastq').__class__):
```

Comment:
allowing fastq will include all subclasses of fastq, which will have different ext, so this is correct. In the previous configuration, where it was specifying fastq and fastqsanger, the fastqsanger was superfluous.